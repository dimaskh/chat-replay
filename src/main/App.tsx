// Libs
import React from 'react'

// Components
import UsersList from 'modules/users/components/UsersList'

// Styles
import 'styles/App.css'

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>Chat Replay Application</p>
      </header>
      <UsersList />
    </div>
  )
}

export default App
