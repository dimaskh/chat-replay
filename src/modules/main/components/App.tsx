// Libs
import React from 'react'

// Components
import Header from 'components/Header'
import Content from './Content'

// Styles
import styled from 'styled-components'

export const App: React.FC = () => (
  <StyledApp>
    <Header>Chat Replay Application</Header>
    <Content />
  </StyledApp>
)

const StyledApp = styled.div`
  display: flex;
  flex-direction: column;
  height: 100vh;
`

export default App
