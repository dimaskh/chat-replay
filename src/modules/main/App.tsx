// Libs
import React from 'react'

// Styles
import 'styles/App.css'

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>Chat Replay Application</p>
      </header>
    </div>
  )
}

export default App
