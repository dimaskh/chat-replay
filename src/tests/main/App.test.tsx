// Libs
import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

// Tested component
import App from 'main/App'

describe('main/App component', () => {
  it('should render correctly', () => {
    const component = shallow(<App />)

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
