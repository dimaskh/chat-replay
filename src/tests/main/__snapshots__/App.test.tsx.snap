// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main/App component should render correctly 1`] = `
<styled.div>
  <Header>
    Chat Replay Application
  </Header>
  <Content />
</styled.div>
`;
